src/components/TableFilterList.js

```diff
@@ -36,16 +39,27 @@ class TableFilterList extends React.Component {
 
     return (
       <div className={classes.root}>
-        {filterList.map((item, index) =>
-          item.map((data, colIndex) => (
+        {filterList.map((item, index) => {
+          if (columnNames[index].filterType === 'custom' && filterListRenderers[index](item)) {
+            return (
+              <Chip
+                label={filterListRenderers[index](item)}
+                key={0}
```

Comment:
Couldn't we still use the `colIndex` for the `key` here?

Reply:
Changed to just use index